extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisorTuningConfig.java

```diff
@@ -169,6 +174,23 @@ public Integer getWorkerThreads()
     return workerThreads;
   }
 
+  @Override
+  public boolean getChatAsync()
+  {
+    if (chatAsync != null) {
+      return chatAsync;
+    } else {
+      return DEFAULT_ASYNC;
+    }
+  }
+
+  @JsonProperty("chatAsync")
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  Boolean getChatAsyncConfigured()
```

Comment:
I did it this way because I didn't want `chatAsync` to appear in serialized tuningConfigs unless it was actually set by the user. And, I wanted us to be able to change the default in the future, and have everyone get the new default if they hadn't explicitly set this. Let me know what you think of that rationale.